assignments/lambda-classes.js

```diff
@@ -1 +1,143 @@
 // CODE here for your Lambda Classes
+class Person {
+    constructor(attb) {
+        this.name = attb.name;
+        this.age = attb.age;
+        this.location = attb.location;
+    }
+
+    speak() {
+        console.log(`Hello my name is ${this.name}, I am from ${this.location}.`)
+    };
+
+}
+
+class Instructor extends Person {
+    constructor(attb) {
+        super(attb); {
+            this.specialty = attb.specialty;
+            this.favLanguage = attb.favLanguage;
+            this.catchPhrase = attb.catchPhrase;
+        }
+    }
+
+    demo(subject) {
+        console.log(`Today we are learning about ${subject}.`);
+    };
+
+    grade(student, subject) {
+        console.log(`${student.name} receives a perfect score on ${subject}`)
+    }
+
+    incDec(student){
+        let min = -5
+        let max = 5
+        student.grade = (student.grade + Math.floor(Math.random() * (max - min) + min))
```

Comment:
Mathematically speaking, What this is doing is a percentage of 10 - 5.  I wonder what the spread of the grades will be